platform/ios/src/MGLMapView.mm

```diff
@@ -564,6 +565,8 @@ - (void)commonInit
     if ([UIApplication sharedApplication].applicationState != UIApplicationStateBackground) {
         [MGLMapboxEvents pushEvent:MGLEventTypeMapLoad withAttributes:@{}];
     }
+
+    [MGLSDKUpdateChecker checkForUpdates];
```

Comment:
`+[MGLOfflineStorage initialize]` or `-[MGLOfflineStorage init]` would also cover the case where the developer uses offline packs early on but buries the map view in a secondary screen. But that's an edge case, so I'm fine with putting it here. Basically we want this check to happen early on while the developer is still paying attention.